### ISSUE TITLE
restoring force ... or not

### ISSUE THREAD

**Issue by JohnDenker** [ericmazur/PnPbook#96]
In section 8.6 on page 184 it says

> Forces exerted by springs are therefore called _restoring forces_.

That's overstated and misleading.  Suggestion:

> > In such a situation, the force exerted by the spring is called a _restoring force_.  (In other situations, such as the mainspring in a windup clock, or in a spear gun, the force exerted by the spring is definitely not a restoring force.).  Conversely, lots of things that aren't springs also provide a restoring force:
> > - Example:  a marble rolling near the bottom of a bowl.
> > - Example:  a weather vane nearly aligned with the wind.

**Comment by ericmazur** [comment]
I agree that there are many more restoring forces than just the forces exerted by springs (and therefore my wording is, most certainly, misleading). However, even the force in a windup clock is "restoring" in the sense that the force is in the direction that will relax the spring. And so is it in the spear gun. Will work on this!

**Comment by JohnDenker** [comment]
If the definition of "restoring force" gets too broad, the word loses all meaning.  I'm not sure what the word is supposed to mean.

In a non-dissipative system, I'm not sure what if anything is being "restored".  Consider a marble rolling in a bowl, in a circular orbit.  In the absence of dissipation, there is no relevant "rest position" or "equilibrium position" or "relaxed condition".  If we want the marble to roll to the bottom _and stay there_ we ought to be talking about entropy, not energy.  See item #155 for a catalog of entropy-related issues.

The same words apply to a mass on a spring.  Suppose the thing is going around in a circular orbit, like a tetherball.  In the absence of dissipation, the spring force is purely transverse.  There is no relevant "relaxed state".

Things get even uglier in the case of a 1/r potential, where it is not even possible to define an equilibrium position, with or without dissipation.

Maybe I'm overlooking something, but when I think of examples where the idea of "restoring force" makes sense, they're all heavily damped.  That makes me suspect that entropy, not energy, is the key to understanding the physics.
